ten/four_test.py

```diff
@@ -1,17 +1,19 @@
 import unittest
 import four_fred
+import four_david
 
 
 class TestTenFour(unittest.TestCase):
 
     def setUp(self):
         self.testcases = [
-            ([(4, 5, 4), (1, 2, 1), (1, 2, 3),
+            ([(4, 5, 4), (1, 2, 1), (1, 2, 4),
```

Comment:
Were you getting bugs with inconsistent order between 123 and 321? I probably should have sorted in the test.